src/users/users.controller.ts

```diff
@@ -87,14 +87,15 @@ export class UsersController {
   @ApiOperation({ summary: '로그인' })
   @UseGuards(LocalAuthGuard) //인터셉터보다 먼저 실행
   @Post('login')
-  async login(@InjectUser() user: User) {
-    return UsersDto.fromEntity(user);
+  login(@Body() content: LoginReqDto) {
+    return content;
```

Comment:
```suggestion
async login(@InjectUser() user: User) {	  
    login(@Body() content: LoginReqDto) {
    return UsersDto.fromEntity(user);
```
커밋 꼬이신거같네요. 다시 돌아왓어요. 이런거 체크 잘 해주셔야해요